roles/sap_vm_provision/defaults/main.yml

```diff
@@ -546,8 +546,91 @@ sap_vm_provision_kubevirt_vm_host_specifications_dictionary:
           mountpoint: /data0
           disk_count: 1                  # default: 1
           disk_size: 512                 # size in GB, integer
-          disk_type: nas                 # KubeVirt Storage Clas
+          disk_type: nas                 # KubeVirt Storage Class
 
+############################################
+# Red Hat OpenShift Virtualization         #
+############################################
+
+# Namespace where the VM should be created in
+sap_vm_provision_ocp_target_namespace: sap
+
+# Username to be created on guest
+sap_vm_provision_ocp_os_user: minion
+
+# Password for the above user
+sap_vm_provision_ocp_os_user_password: banana
```

Comment:
It would be safer to not provide defaults for passwords. What do you think?

Reply:
Agreed, should be removed.